Headers/AppKit/AppKit.h

```diff
@@ -216,6 +216,7 @@
 #import <AppKit/NSStepperTouchBarItem.h>
 #import <AppKit/NSStepper.h>
 #import <AppKit/NSStepperCell.h>
+#import <AppKit/NSSwitch.h>
```

Comment:
Should we include NSAccessibilityProtocols.h here as well?

Reply:
I believe so.